src/prompts/alert/alert.scss

```diff
@@ -0,0 +1,114 @@
+@import '../common.scss';
+
+.kumulos-alert-container {
+    transition: transform 0.2s $quintEasing, box-shadow 0.2s $quintEasing;
+    will-change: transform, box-shadow, width, height;
```

Comment:
Looks like these might be copied from the Bell styles, but not convinced these will apply to the alert?

Reply:
Residual - will remove!